main.css

```diff
@@ -37,45 +39,59 @@ html, body
 #header
 {
     position: fixed;
-    height: 30px;
+    height: 45px;
     width: 100%;
     color: white;
     box-sizing: border-box;
-    background: #0D2A52;
+    background: #00254A; /* color picked from Pitt website */
+    line-height: 45px;
+}
+
+#header a{
+  color: white;
 }
 
-#header h3
+#logo{
+  float: left;
+  margin: 0 10px 0 15px;
+  max-width: 45px;
+  max-height: 45px;
+}
+
+#header h1
 {
```

Comment:
Changing the header title to an `h1` helps with the SEO, but the default CSS nudges out the About navigation link on mobile. Setting `font-size` to to `18px` fixes this. There's probably a cleaner fix, but I can't think of any other links that we need to add that would break it.